src/views/v2/events-bar/views/item.php

```diff
@@ -23,24 +23,22 @@
 $view_instance = View::make( $this->get( 'view_class_name' ) );
 $view_slug = $view_instance->get_slug();
 $is_current_view = $view->get_slug() === $view_instance->get_slug();
-$view_url = tribe_events_get_url( [ 'eventDisplay' => $view_instance->get_slug() ], $this->get( 'view' )->get_url() );
+$view_url = tribe_events_get_url( [ 'eventDisplay' => $view_slug ], $this->get( 'view' )->get_url() );
+
+$link_classes = [ 'tribe-events-c-view-selector__list-item-link' ];
+if ( $is_current_view ) {
+	$link_classes[] = 'tribe-events-c-view-selector__list-item-link--active';
+}
 ?>
-<li class="tribe-common-form-control-tabs__list-item" role="presentation">
-	<input
-		class="tribe-common-form-control-tabs__input"
-		id="<?php echo sanitize_html_class( "tribe-views-{$view_slug}" ); ?>"
-		name="tribe-views"
-		type="radio"
-		value="<?php echo esc_attr( $view_slug ); ?>"
-		<?php checked( $is_current_view ); ?>
-	/>
-	<label
-		class="tribe-common-form-control-tabs__label"
-		id="<?php echo sanitize_html_class( "tribe-views-{$view_slug}-label" ); ?>"
-		for="<?php echo sanitize_html_class( "tribe-views-{$view_slug}" ); ?>"
-		role="option"
-		aria-selected="true"
+<li class="tribe-events-c-view-selector__list-item">
```

Comment:
I think it'd also be cool to know if the item is the one active. Thinking about customizations or future amendments.

Reply:
The link below this has an active class. Are you talking about the list item getting an active class?

Reply:
Yes. As it's the item the one that will be active.

Reply:
Got it. My preference would be to not, as the `<li>` elements here are basically semantic wrappers. They don't do much of anything at all, the main actors here are the links.

Reply:
If customers want to add an active wrapper class for styling, it's not difficult to extend what's already here.

Reply:
Yes, my idea was basically that if they want to do it, it would be a small effort for us now and they will have to add some CSS instead of having to touch the template.

Reply:
I understand your point, I don't see the benefit to us to do so other than an active modifier class that doesn't do anything style-wise or functionality-wise. I believe we should provide customers the tools to do what they want, but I don't believe we should try to provide them everything, whether they will use them or not.

Reply:
moved active class to list item and removed from anchor.